build.gradle

```diff
@@ -21,6 +21,7 @@ allprojects {
 // we need a repository for the combination of jacoco and sonarcloud
 repositories {
   mavenCentral()
+  maven { url "https://oss.sonatype.org/content/repositories/snapshots" }
```

Comment:
Do we still need this?

Reply:
No. I already removed it with the last force push but maybe missed something. Should be fixed now.